src/zkevm_specs/state.py

```diff
@@ -375,36 +375,49 @@ def check_account_destructed(row: Row, row_prev: Row):
 def check_tx_log(row: Row, row_prev: Row):
     # tx_id | log_id | field_tag | index | value
     tx_id = row.id()
-    pre_tx_id = row_prev.id()
+    prev_tx_id = row_prev.id()
     log_id = row.address()
-    pre_field_tag = row_prev.field_tag()
+    prev_log_id = row_prev.address()
```

Comment:
yes, the spec don't update to match circuit side for log id lookup related, because  current circuit side implementation have shortage regarding (log_id + index ) as address.   I have created an issue #224 to describe it,  have to update it in future anyway.

Reply:
I would suggest resolving https://github.com/privacy-scaling-explorations/zkevm-specs/pull/227 , and once it's merged, update this PR (if necessary), and then from my side this PR will look good :)
Let me know if you'd like to proceed in a different way.

Reply:
I would rebase https://github.com/privacy-scaling-explorations/zkevm-specs/pull/227 into this PR as it is simple .